docs/secure-connections/sauce-connect-5/advanced/specifications.md

```diff
@@ -0,0 +1,30 @@
+---
+id: specifications
+title: Sauce Connect 5 Specifications
+sidebar_label: Specifications
+---
+
+## Supported Browsers and Ports {#sc5browsers}
```

Comment:
How about creating a new directory `specifications` and put it there. I'd like to migrate away from `advanced` as it's too generic and it seems to have no advantages. 

Supported browsers is also misleading name as we (sauce connect team) won't maintain a list of supported browsers in sauce. It's only the list of ports that are set to proxy localhost traffic. How about `localhost-ports`?